src/main/java/com/example/dto/UserLoginRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package com.example.dto;
+
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+
+public record UserLoginRequestDto(
+        @NotEmpty
```

Comment:
for String use `@NotBlank`
```suggestion
        @NotBlank
```